test/integration/02-user-wallet/02-receive-onchain.spec.ts

```diff
@@ -95,745 +119,1540 @@ afterAll(async () => {
   await bitcoindClient.unloadWallet({ walletName: "outside" })
 })
 
-describe("FunderWallet - On chain", () => {
-  it("receives on-chain transaction", async () => {
-    const funderWalletId = await getFunderWalletId()
-    await sendToWalletTestWrapper({
-      walletId: funderWalletId,
-      amountSats: getRandomAmountOfSats(),
-    })
-  })
-})
+describe("With Bria", () => {
```

Comment:
didnt we say that this should have a new test file?

Reply:
This file is pretty much already all bria. We will remove the `With Lnd` describe block once we switch to handling lnd xpubs from bria